test/tokenbridge.test.ts

```diff
@@ -45,61 +45,91 @@ const USDC_WHALE = "0x47ac0fb4f2d84898e4d9e7b4dab3c24507a6d503";
 describe("TokenBridge", async function () {
   this.timeout(TIMEOUT);
 
+  const networkUrl: string = (network.config as HttpNetworkConfig).url;
+  const abiCoder = new ethers.utils.AbiCoder();
+
+  // users
   let proxyAdmin: SignerWithAddress;
+
   let l1user: SignerWithAddress;
+  let l2owner: Account;
   let l2user: Account;
   let signer: SignerWithAddress;
   let daiWhale: providers.JsonRpcSigner;
   let usdcWhale: providers.JsonRpcSigner;
   let stkaaveWhale: providers.JsonRpcSigner;
-  const networkUrl: string = (network.config as HttpNetworkConfig).url;
-  console.log(networkUrl);
-  const abiCoder = new ethers.utils.AbiCoder();
+
+  // misk
   let blockNumberDai: number;
   let blockNumberUsdc: number;
-  let l1tokenDaiInitialBalance: number;
-  let l1tokenUsdcInitialBalance: number;
   let txDai: any;
   let txUsdc: any;
+
   // L2
-  let L2TokenFactory: StarknetContractFactory;
-  let l2tokenDai: StarknetContract;
-  let l2tokenUsdc: StarknetContract;
-  let proxyL2TokenDai: StarknetContract;
-  let proxyL2TokenUsdc: StarknetContract;
-  let proxiedL2TokenDai: StarknetContract;
-  let proxiedL2TokenUsdc: StarknetContract;
-  let TokenBridgeL2: StarknetContractFactory;
-  let tokenBridgeL2: StarknetContract;
-  let proxiedRewAaveTokenL2: StarknetContract;
-  let ProxyFactoryL2: StarknetContractFactory;
-  let proxyTokenBridgeL2: StarknetContract;
-  let proxiedTokenBridgeL2: StarknetContract;
-  let rewAaveTokenImplementation: StarknetContract;
-  let proxyRewAAVEToken: StarknetContract;
+  //// factories
+  let l2TokenFactory: StarknetContractFactory;
+  let l2ProxyFactory: StarknetContractFactory;
+  let l2TokenBridgeFactory: StarknetContractFactory;
+
+  //// tokens
+  let l2StaticADaiImpl: StarknetContract;
+  let l2StaticADaiProxy: StarknetContract;
+  let l2StaticADai: StarknetContract;
+
+  let l2StaticAUsdcImpl: StarknetContract;
+  let l2StaticAUsdcProxy: StarknetContract;
+  let l2StaticAUsdc: StarknetContract;
+
+  let l2rewAAVEImpl: StarknetContract;
```

Comment:
there will be some conflicts with #60 (should we use `Impl` instead of the whole word?)